src/nvim/highlight_group.c

```diff
@@ -173,15 +173,13 @@ static const char *highlight_init_both[] = {
   "default link PmenuKindSel   PmenuSel",
   "default link PmenuSbar      Pmenu",
   "default link Substitute     Search",
-  "default link TabLine        StatusLine",
+  "default link TabLine        StatusLineNC",
```

Comment:
Comparing `hi link Tabline StatusLineNC` (top) to `hi link TabLine StatusLine` (bottom):

<img width="482" alt="image" src="https://github.com/neovim/neovim/assets/1359421/c079fb82-e1ca-4fa2-8f1f-05097674f60c">

I can see an argument against for keeping the tabline more "dimmed" since it has less reason to stand-out than the actual statusline (which signals "current window"). But when it is linked to `StatusLine` it is confused with `CurrentLine`, so I can't easily tell if I _have any tabs open_ or if that's just my cursorline at the top of the window.

Reply:
Yes, unfortunately, that's true. The reason being that there are just (deliberately) not that many shades of grey. The reasonable choices are:
1. The `Nvim*Grey1` which is darkest/litest and which essentially was before. Problem: might be too similar to `Normal` as it was one of the reasons to change statusline colors. Also it is not linked to semantically proper group and would need extra adjusting from user.
2. `Nvim*Grey2`. Problem - same as `Normal`.
3. `Nvim*Grey3`. The one which is essentially now. Problem: same as `Cursorline`.
4. `Nvim*Grey4`. Problem: same as `Visual` and has less contrast ratio.
5. Values from other lightness. Problem - too stand out.

Weighing all this, decided that linking to `StatusLineNC` and having occasional "merged" tabline and current line is the best compromise.